lookout/style/typos/general.py

```diff
@@ -0,0 +1,83 @@
+from typing import Any, Mapping, Optional
+
+from google_drive_downloader import GoogleDriveDownloader as gdd
+import pandas
+
+from lookout.style.common import merge_dicts
+from lookout.style.typos.preprocessing import filter_splits, print_frequencies
+from lookout.style.typos.utils import Columns, flatten_df_by_column
+
+
+DRIVE_DATASET_ID = "1muNVWPe68XK8SFvqIv3V728NmkT46aTx"
```

Comment:
These should be the defaults overridable in function arguments. `DEFAULT_DRIVE_DATASET_ID`